### PR TITLE
Friendly commit to see if you're collaborating.

### PR DESCRIPTION
**Pull request by Spatbee**
Added a filter for .log files, added to tsconfig to make local setup easier

### CODE REVIEW

File: src/Importer.tsx [hunk 1]
```diff
@@ -91,6 +91,7 @@ function Importer({ show, onImport, onClose }: ImportProps): JSX.Element {
               <Form.Control
                 type="file"
                 aria-describedby="logFileHelp"
+                accept=".log"
```
Comment (glenjamin) [inline src/Importer.tsx:94]: oh nice, TIL!

File: tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@tsconfig/create-react-app/tsconfig.json",
   "compilerOptions": {
+    "jsx": "react-jsx",
```
Comment (glenjamin) [inline tsconfig.json:4]: Do you have a docs link for what this does? Things seemed to be working ok for me in locally dev with VSCode

**Review by glenjamin — APPROVED** [review]

File: tsconfig.json (hunk 2 above)
Comment (Spatbee) [reply tsconfig.json:4]: My VSC would not compile/lint without it
https://www.typescriptlang.org/tsconfig#jsx
https://stackoverflow.com/a/65539274/5093905

Comment (glenjamin) [reply tsconfig.json:4]: hrm, I wonder it worked for me 🤷

**Pull request merged**